container-builder/builder/kubernetes/testdata/Dockerfile

```
@@ -32,7 +32,7 @@ RUN ${MAVEN_HOME}/bin/mvn -U -B ${MAVEN_ARGS_APPEND} -s ${MAVEN_SETTINGS_PATH} c
 #=============================
 # Runtime Run
 #=============================
-FROM registry.access.redhat.com/ubi8/openjdk-11:1.11
+FROM registry.access.redhat.com/ubi8/openjdk-17:1.18-2
```

use abstract 1.18 tag instead of specific 1.18-2 , in case another 1.18 build or CVE fix, the latest will be used and image up to date. 
```suggestion
FROM registry.access.redhat.com/ubi8/openjdk-17:1.18
```